### ISSUE TITLE
Status of Repo

### ISSUE THREAD

**Issue by LewisLarsen** [rearley/ebay#6]
Hi,

Is this project still maintained? Looking for a PHP library for the eBay API, primarily to search items for a Laravel project of mine.

If it is no longer something you're supporting, no worries but I wanted to double-check before I tried it out. 

Thanks,
Lewis

**Comment by rearley** [comment]
@LewisLarsen no, unfortunately I have let this REPO go unmaintained for awhile due to other projects. I keep telling myself I want to breath some new life into it, but not sure when that will occur. I would go with @yiiprogrammer suggestion.
